III-6036 Add paragraph about completeness advanced query (#367)
cultuurnet/apidocs: projects/uitdatabank/docs/search-api/advanced/advanced-queries.md

GET /offers/?q=videosCount:[5 TO 10]
```

### completeness

> 

Reviewer: ```suggestion
<!-- theme: info -->

> #### New functionality 🚧
>
> `completeness` is a new parameter that is still under construction. At the moment the completeness is only calculated for events, places and organizers created after February 15, 2024. 
```

Reviewer: I would add this disclaimer so we can release the documentation for the parameter. We can then remove it once the dataset of UiTdatabank has been replayed

> With the `completeness` field you can filter events, places and organizers by their completeness score. The completeness score is a number between 0 and 100 that indicates how complete the information of an event, place or organizer is.

Reviewer 2: Completely out of scope and probably too confusing for users of the documentation, but I just realized that since certain fields are mandatory, the completeness score can never be lower than 36/48

Reviewer 3: I think we still have some invalid JSON projections that have a lower score. A replay should fix those.

**Applicable on endpoints**

`/events` `/places` `/offers` `/organizers`

**Possible values**

* an integer, e.g. `50`
* a range, consisting of two integers (lower and upper bound), e.g. `[50 TO *]`

**Examples**

Retrieve all events and places that have at least a completeness score of 50:

```
GET /offers/?q=completeness:>=50
```

Retrieve all events and places that have completeness score of exactly 65:

```
GET /offers/?q=completeness:65
```

> Retrieve all events and places that have a completeness score less than 50:

Reviewer: ```suggestion
Retrieve all organizers that have a completeness score less than 50:
```

```

> GET /offers/?q=completeness:<50

Reviewer: ```suggestion
GET /organizers/?q=completeness:<50
```

Reviewer: Nit: just to indicate it is also possible to use on the `/organizers` endpoint
